cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -2417,35 +2481,35 @@ private void clearSubscriptionData() {
         sharedPreferences.edit().remove(CleverPushPreferences.SUBSCRIPTION_ATTRIBUTES).apply();
     }
 
-	public boolean areAppBannersDisabled() {
-		SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(CleverPush.context);
-		appBannersDisabled = sharedPreferences.getBoolean(CleverPushPreferences.APP_BANNERS_DISABLED, false);
-		return appBannersDisabled;
-	}
+    public boolean areAppBannersDisabled() {
```

Comment:
please refactor to "isAppBannersDisabled()". 

"are" is not part of the java naming conventions. 
see https://docs.oracle.com/javase/tutorial/javabeans/writing/properties.html

(boyscout) 🔥